packages/contracts/contracts/Unirep.sol

```diff
@@ -128,11 +128,10 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
     }
 
     function verifyNullifier(uint256 nullifier) private {
```

Comment:
@vivianjeng what do you think of adding `require(nullifier != 0)` here? It might protect against zero value nullifiers accidentally being passed in.